### PR TITLE
feat: upgrade docker version

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: python/openmldb_tool/setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='openmldb-tool',
-    version='0.7.0a0',
+    version='0.8.2',
```
Comment (aceforeverd) [inline python/openmldb_tool/setup.py]: keep the `a0`, version number in main is pre-release, not final number

File: steps/prepare_release.sh
Comment (aceforeverd) [inline steps/prepare_release.sh:1]: If the change is about the py number, this change is unnecessary since https://github.com/4paradigm/OpenMLDB/pull/3388/files#r1270313819

Comment (dl239) [reply steps/prepare_release.sh:1]: change the version of CMAKE, python, java

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: steps/prepare_release.sh [hunk 2]
```diff
@@ -69,6 +72,9 @@ if [[ ${#ARR[@]} -gt 3 ]]; then
     # refer: https://www.python.org/dev/peps/pep-0440/#pre-releases
     PY_VERSION="${ARR[0]}.${ARR[1]}.${ARR[2]}${ARR[3]}"
 fi
+if [[ ${VERSION} =~ "SNAPSHOT" ]]; then
+    PY_VERSION="${ARR[0]}.${ARR[1]}.${ARR[2]}a0"
```
Comment (aceforeverd) [inline steps/prepare_release.sh]: conflict with line 73. I intend to run as :
```sh
./prepare_release.sh 1.2.3.a0
```

then the `PY_VERSION` is `1.2.3a0`. For cpp, `.a0` is not token; for java, `.a0` always replaced with `-SNAPSHOT`.  See the comment at top of the file. 

See if you require more semantic, or we can revert the change.

**Review by aceforeverd — APPROVED** [review]
